python/interpret_community/explanation/explanation.py

```diff
@@ -485,6 +485,15 @@ def get_raw_feature_importances(self, raw_to_output_maps):
         else:
             return _get_raw_feature_importances(np.array(self.local_importance_values), raw_to_output_maps).tolist()
 
+    def _perf_dict(self, y, y_hat, i):
```

Comment:
Will you add a unit test or two for this?

Reply:
we already have e2e unit tests and they caught this, the problem was we were testing against the older version of interpret-core

Reply:
I can add one in separate PR